_includes/v1.0/faq/clock-synchronization-monitoring.html

```diff
@@ -5,4 +5,4 @@
 `clock_offset_meannanos` | The mean difference between the node's clock and other nodes' clocks in nanoseconds
 `clock_offset_stddevnanos` | The standard deviation of the difference between the node's clock and other nodes' clocks in nanoseconds
 
-As described in [the above answer](#what-happens-when-node-clocks-are-not-properly-synchronized), a node will kill itself if the mean offset of its clock from the other nodes' clocks exceeds 80% of the maximum offset allowed. It's recommended to monitor the `clock_offset_meannanos` metric and alert if it's approaching the 80% threshold of your cluster's configured max offset.
+As described in [the above answer](#what-happens-when-node-clocks-are-not-properly-synchronized), a node will shutdown if the mean offset of its clock from the other nodes' clocks exceeds 80% of the maximum offset allowed. It's recommended to monitor the `clock_offset_meannanos` metric and alert if it's approaching the 80% threshold of your cluster's configured max offset.
```

Comment:
I think the verb needs to be 2 words "shut down"

Reply:
Fixed